builtin/providers/aws/resource_aws_workspace.go

```diff
@@ -0,0 +1,246 @@
+package aws
+
+import (
+	"fmt"
+	"log"
+	"strings"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/workspaces"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceAwsWorkspace() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsWorkspaceCreate,
+		Read:   resourceAwsWorkspaceRead,
+		Update: resourceAwsWorkspaceUpdate,
+		Delete: resourceAwsWorkspaceDelete,
+
+		Schema: map[string]*schema.Schema{
+			"bundle_name": &schema.Schema{
+				Type:     schema.TypeString,
+				ForceNew: true,
+				Required: true,
+			},
+
+			"bundle_id": &schema.Schema{
```

Comment:
If you don't set this and then it get's set in the Read func, you will get a perpetual diff. As this is 'ForceNew: true', that could be problematic. I think this will also need to be `Computed: true`